bootcamp/tutorials/integration/llama_agents_metadata.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<p>this part is no longer download data.</p><p> </p><p>something simple like:</p><p> </p><p>Prepare Embedding Model</p><p> </p><p>We use open-source embedding model xxx listed on Hugging Face xxxx</p><p>-- code --</p><p> </p><p>should do the job.</p><p> </p><p>Same idea applies to the following sections.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1401/discussion/#comment-1705463981'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>